src/analysis/normed_space/hahn_banach/separation.lean

```diff
@@ -186,3 +186,16 @@ begin
       (convex_singleton y) is_closed_singleton (disjoint_singleton.2 hxy),
   exact ⟨f, by linarith [hs x rfl, ht y rfl]⟩,
 end
+
+/-- **Eidelheit's Theorem** -/
```

Comment:
Do you have a reference for this name? Putting in the PR description is probably sufficient.

Reply:
I cannot find the reference anymore and most times Eidelheit's Theorem refers to a related but different theorem. 😢

I still think it's worth naming it over it being an anonymous lemma, in case someone is looking for it.

Reply:
If you can't locate any reference using this name, I think it's better to just add a docstring explaining what the theorem is saying, but removing the name.